README.rst

```diff
@@ -46,7 +46,21 @@ To build the PDF form of the documentation simply execute the following command:
 
   make latexpdf
 
-Note: Creating the PDF form of the documentation requires that latexmk is available, information on latexmk can be found at https://ctan.org/pkg/latexmk/.
+Note: Creating the PDF form of the documentation requires that some or all of a tool like Windows/MikTeX, macOS/MacTeX, Windows/TeXlive, Linux/TeXlive is available to the Sphinx build tool.
 
-Note: The documentation built with `make html` is different to the documentation built with `make latexpdf`.
-The HTML form of the documentation is the full form being both the normative specification and the informative specification, the PDF form of the documentation is solely the normative specification.
+Documentation builds
+^^^^^^^^^^^^^^^^^^^^
+
+There are three types of build that can be created from this codebase.
+
+1. Full documentation build of the normative and informative specification combined.
```

Comment:
Might want to mention that it's the default type?

Reply:
(or remove until it's ready for PDF printing ...?)